src/components/Button.js

```diff
@@ -2,14 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
```

Comment:
props should contain color and wide, ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)